### DISCUSSION TITLE
ERR Error during retention scan error="open /var/local/inbucket/mail/cfc/cfc569: too many open files" module=storage

### DISCUSSION THREAD

**Discussion by techsachin1** [inbucket/inbucket#371] [Q&A]
when i check status of Inbucket systemctl status inbucket shows this error

**Comment by jhillyerd** [comment]
You could try adding lines to increase the file limit to inbucket.service, see https://unix.stackexchange.com/questions/623599/systemd-user-increase-max-open-files and similar

**Reply by techsachin1** [reply]
i didnt understand which lines i have to add in inbucket.service

**Comment by Kasztan404** [comment]
in my case it was (invalid exec path):

● inbucket.service - Inbucket Disposable Email Service

   Loaded: loaded (/lib/systemd/system/inbucket.service; enabled; vendor preset: enabled)

   Active: failed (Result: exit-code) since Fri 2023-05-26 08:45:16 CEST; 7s ago

  Process: 567 ExecStartPre=/bin/mkdir -p /var/local/inbucket (code=exited, status=0/SUCCESS)

  Process: 568 ExecStartPre=/bin/chown daemon:daemon /var/local/inbucket (code=exited, status=0/SUCCESS)

  Process: 569 ExecStart=/usr/local/bin/inbucket (code=exited, status=203/EXEC)

 Main PID: 569 (code=exited, status=203/EXEC)

 

 > so I had to correct it (In my opinion it is bug):

 pico /lib/systemd/system/inbucket.service

 

 edit this:

 /usr/local/bin/inbucket

 

 to that:

 /usr/bin/inbucket


> save file

 

> reload systemctl:

systemctl daemon-reload

systemctl start inbucket


> check again:

systemctl status inbucket
